tests/common.c

```diff
@@ -716,9 +721,7 @@ parsec_context_t* setup_parsec(int argc, char **argv, int *iparam)
         }
     }
     if( nbgpu > 0 ) {
-        cublasStatus_t status = cublasInit();
-        assert(CUBLAS_STATUS_SUCCESS == status);
-        parsec_info_register(&parsec_per_stream_infos, "DPLASMA::CUDA::HANDLES",
+        CuHI = parsec_info_register(&parsec_per_stream_infos, "DPLASMA::CUDA::HANDLES",
                              destroy_cuda_handles, NULL,
```

Comment:
why one of these function is name-spaced with dplasma while the other one is not ? Why is the info called `parsec_per_stream_infos` when it clearly belongs to dplasma. I know there issues are not related to this PR, but I just noticed them while reading through the code.